File: src/main/java/historywowa/domain/heritage/domain/repository/HeritageRepository.java
package historywowa.domain.heritage.domain.repository;

import historywowa.domain.heritage.domain.entity.Heritage;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface HeritageRepository extends JpaRepository<Heritage, Long> {
}

File: src/main/kotlin/historywowa/domain/heritage/domain/repository/HeritageRepository.kt
package historywowa.domain.heritage.domain.repository

import historywowa.domain.heritage.domain.entity.Heritage
import historywowa.domain.member.domain.entity.Member
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface HeritageRepository : JpaRepository<Heritage, Long> {

    fun findByMember(member: Member): List<Heritage>
}
